venafi/provider.go

```diff
@@ -1,19 +1,21 @@
 package venafi
 
 import (
+	"context"
```

Comment:
could we organize the imports as the following: golang packages, then this project packages, then our company other packages, and in the end the 3rd party packages. this is just a suggestion

Reply:
From Golang version 1.17 (the one we use for building our Terraform binaries) `context` it is now a built-in library